doc/uploading-files.md

```diff
@@ -0,0 +1,55 @@
+## Uploading files
+
+Lets say you want your blog posts to contain images.
+Here's how you add file-upload in `data/collection/post.json`:
+
+    "properties":{
+
+      ...
+
+      "picture": {
+          "title": "Picture",
+          "type": "string",
+          "media": {
+              "binaryEncoding": "base64",
+              "type": "image/png"
+          },
+          "links": [
+             {
+               "href": "/custom_endpoint/{{self}}.png",
+               "mediaType": "image/png"
+             }
+          ]
+      }
+
+Or generic files:
+
+    "properties":{
+
+      ...
+
+      "file": {
+        "type": "string",
+        "format": "file",
+        "title": "File"
+        "links": [
+          {
+            "rel": "Download File",
+            "href": "/custom_endpoint/{{self}}",
+            // Can also set `download` to a string as per the HTML5 spec
+            "download": true
+          }
+        ]
+      }
+
+> NOTE: the links probably will need a custom endpoint which serves the file with the proper media type 
+
+    expressa.get('/files/:file',function(req,res,next){
```

Comment:
What do you think about recommending express.static here instead?
https://expressjs.com/en/starter/static-files.html